app/policies/application_policy.rb

```diff
@@ -53,7 +53,8 @@ def resolve
   end
 
   protected
-    def owner_or_admin_with_access?
-      (member.admin? && record.role != 'owner') || member.owner?
-    end
+
+  def owner_or_admin_with_access?
+    (member.admin? && record.role != 'owner') || member.owner?
```

Comment:
@yunixon why you check `role` for `record`? `record` may be any model and may have not `role` field.